pkg/jsonnet/imports.go

```diff
@@ -56,7 +57,14 @@ func TransitiveImports(dir string) ([]string, error) {
 
 	paths := make([]string, 0, len(imports)+1)
 	for k := range imports {
-		paths = append(paths, k)
+
+		// Try to resolve any symlinks; use the original path as a last resort
+		p, err := filepath.EvalSymlinks(k)
+		if err != nil {
+			log.Fatalln("Failed to resolve symlink: ", err)
```

Comment:
Packages other than `main` should not call `os.Exit()`, as this skips `defer`, etc. Instead, please return the error:

```go
return nil, errors.Wrap(err, "resolving symlinks")
```

Reply:
Fixed.